tslint.json

```diff
@@ -3,6 +3,7 @@
 		"build/lib/tslint"
 	],
 	"rules": {
+		"no-string-throw": true,
```

Comment:
This one should probably not even use a `throw`. In this PR, you can either leave it as is or convert it to something like `throw new Error('Could not resolve codelens')`